app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -36,8 +38,31 @@ def get_content(link)
       client.repository(user_name + "/" + repo_name)
     end
 
+    def valid_option(option)
+      option.match(/no-readme/)
+    end
+
+    def parse_options(link)
+      opts = sanitize_link(link)
+      _, *options = opts.split(" ")
+
+      validated_options = options.map { |option| valid_option(option) }.reject(&:nil?)
+      raise StandardError, "Invalid Options" unless options.empty? || !validated_options.empty?
```

Comment:
I think using `any?` here would be more semantic
```suggestion
      raise StandardError, "Invalid Options" unless options.empty? || validated_options.any?
```